### DISCUSSION TITLE
A question about font rendering on Spiral

### DISCUSSION THREAD

**Discussion by behrangsa** [SpiralLinux/SpiralLinux-project#313] [General]
Hi all,

By "font rendering and color theming preconfigured for optimal legibility", do you mean that you have just disabled bitmap fonts? Or have you made other font-related optimizations too?

**Comment by geckolinux** [comment]
Hi there, thanks for the question.

I have added and enabled the following low-level font configuration files in `/etc/fonts/conf.d` :
```
11-lcdfilter-default.conf
20-no-embedded.conf
42-antialias.conf
68-color-emoji.conf
```

The desktop environments are also all configured to use slight hinting with RGB antialiasing.

Additionally, I realized that the Flatpak environment by default only uses grayscale antialiasing even if the host desktop uses RGB antialiasing, so I forced RGB for Flatpaks with the `~/.config/fontconfig/fonts.conf` file and added an override to the global Flatpak configuration to make it use that configuration.

And although it might seem relatively simple, the actual font family chosen (Ubuntu) is subjectively much better than other default fonts for legibility and on-screen rendering. Also the GTK and Qt themes have been carefully chosen to offer good contrast between the font color and the background element colors, which is often overlooked in many trendy and/or default themes.
